pgbot/__init__.py

```diff
@@ -175,6 +176,20 @@ async def message_delete(msg: discord.Message):
                     del common.cmd_logs[log]
                     return
 
+    if common.GENERIC:
+        return
+
+    if msg.channel in common.entry_channels.values():
+        history = await common.entries_discussion_channel.history(
+            around=msg.created_at
```

Comment:
It also probably needs a `limit=1` kwarg over here, so that we do not waste time iterating over 1000 other messages

Reply:
Usually there wouldn't be 1000 messages returned because uploads to those channels are rare so i thought limit wouldn't be needed. I had thought about adding the limit and now that i think about it, there's no harm in it.